third_party/terraform/utils/config.go.erb

```diff
@@ -83,6 +83,7 @@ type Config struct {
 	AccessContextManagerBasePath string
 	CloudSchedulerBasePath       string
 	FirestoreBasePath            string
+  MLEngineBasePath             string
```

Comment:
It'll get gofmtted downstream so it's not a big deal, but it would certainly look nicer in an editor if these were tabs (and in the other files too)